backend/src/main/java/ai/verta/modeldb/batchProcess/DatasetToRepositoryMigration.java

```diff
@@ -272,7 +272,15 @@ private static void migrateDatasetVersionToCommitsBlobsMigration(
             try {
               DatasetVersion newDatasetVersion = datasetVersionEntity.getProtoObject();
               if (newDatasetVersion.hasPathDatasetVersionInfo()) {
-                createCommitAndBlobsFromDatsetVersion(session1, newDatasetVersion, repoId);
+                String commitHash =
+                    createCommitAndBlobsFromDatsetVersion(session1, newDatasetVersion, repoId);
+
```

Comment:
```suggestion
LOGGER.debug("{} datasetversion mapped to {} commit", newDatasetVersion.getId(), commitHash);
```

This might not be syntactically correct so please add the log and verify. the idea is to have some audit trail in case we want to revert back

Also add similar log for dataset and repo mapping